src/deploy/functions/prompts.ts

```diff
@@ -19,33 +19,30 @@ import { Options } from "../../options";
  */
 export async function promptForFailurePolicies(
   options: Options,
-  want: backend.FunctionSpec[],
-  have: backend.FunctionSpec[]
+  want: backend.Backend,
+  have: backend.Backend
 ): Promise<void> {
   // Collect all the functions that have a retry policy
-  const retryFunctions = want.filter((fn) => {
-    return backend.isEventTrigger(fn.trigger) && fn.trigger.retry;
+  const retryEndpoints = backend.allEndpoints(want).filter((fn) => {
+    return backend.isEventTriggered(fn) && fn.eventTrigger.retry;
```

Comment:
```suggestion
  const retryEndpoints = backend.someEndpoint(wantBackend, (e) => {
    return backend.isEventTriggered(e) && e.eventTrigger.retry;
```

Reply:
I'll change the parameter name, but `allEndpoints.filter` is intentional. we're getting a list, not a boolean.